examples/kubernetes/cilium-metrics-svc.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: cilium-metrics-svc
+  namespace: kube-system
+  labels:
+    k8s-app: cilium-metrics-svc
+    app: cilium-metrics-svc
+spec:
+  type: NodePort
```

Comment:
Why NodePort?

Reply:
I think this is my fault. When I did the prometheus examples I used a nodeport (https://github.com/cilium/cilium/blob/master/examples/kubernetes/prometheus.yaml#L9-L27).

Looking at it now, knowing more about stuff, I think we can build it with a normal service, and not a nodeport (since the prometheus collector and server are running in a pod anyway).

Reply:
My bad; this should be clusterIP. Thanks.